main.py

```diff
@@ -35,107 +49,105 @@ def main():
     if args.channels != 1 and args.channels != 3:
         print('Invalid number of channels. Must be (1) for black/white or (3) for RGB')
 
-    # Text generation
     if args.script_file:
         with open(args.script_file, 'r') as f:
             script = f.read()
         print('script_file provided, skipping text generation')
-
     else:
         meditation_types = ['focused', 'body-scan', 'visualization', 'reflection', 'movement']
         if args.med_type not in meditation_types:
-            print('Invalid input. Provide valid med_type or input own script. Exitting...')
-            sys.exit()
-        script = text_generation(args.med_type)
-
-        script_file_name = f'{args.med_type}-meditation-script1.txt'
+            raise Exception('Invalid input. Provide valid med_type or input own script. Exitting...')
+
+        prompts = {
+            'focused': 'write me a focused meditation script designed to enhance focus and attention by noticing all 5 senses',
+            'body-scan': 'write me a body scan meditation script to relax and relieve stress by tightening and relaxing muscles',
+            'visualization': 'write me a visualization meditation script noticing all 5 senses at the beach/garden and is designed to boost mood, reduce stress, and promote inner peace',
+            'reflection': 'write me a reflection meditation script designed to increase self awareness, mindfulness, and gratitude by asking the user about the current day and the recent past',
+            'movement': 'write me a movement meditation script designed to improve mind body connection, energy, vitality, and the systems of the body'
+        }
+        prompt = prompts[args.med_type]
+
+        ###################
+        # Text generation #
+        ###################
+        script = text_generation(prompt)
+        script_file_name = f'data/{args.med_type}_meditation_script.txt'
         try:
             with open(script_file_name, 'w') as f:
                 f.write(script)
             print(f'Storing meditation script in {script_file_name}')
+        except ValueError as error:
+            print(f"Couldn't store script in file {script_file_name}: {error}")
+
+    ##################
+    # Text-to-speech #
+    ##################
+    if args.skip_audio_generation:
```

Comment:
For both skip_audio_generation and skip_cppn_generation, you put store_false as the action above. I personally think that it's more intuitive to **store_true** so that here we can say "if **not** skip_audio_generation" below, then generate the audio

Reply:
I think the code was not 100% updated. I have left only skip and it is the `skip_cppn_generation`. As default, it is store_true as you suggested.

Reply:
If it helps you, feel free to add more skips!

Reply:
Did this just leaving store_false so it is running as default. Feel free to change it though if you feel more comfortable.
Also, the two skips there are right now are:
- skip_cppn_generation
- skip_background_music